app/celery/nightly_tasks.py

```diff
@@ -69,6 +74,39 @@ def _remove_csv_files(job_types):
         current_app.logger.info("Job ID %s has been removed from s3.", job.id)
 
 
+@notify_celery.task(name="archive_unsubscribe_requests")
+def archive_unsubscribe_requests():
+    for service_id in get_service_ids_with_unsubscribe_requests():
+        archive_batched_unsubscribe_requests.apply_async(queue=QueueNames.REPORTING, args=[service_id])
+        archive_old_unsubscribe_requests.apply_async(queue=QueueNames.REPORTING, args=[service_id])
+
+
+@notify_celery.task(name="archive_batched_unsubscribe_requests")
+def archive_batched_unsubscribe_requests(service_id):
+    start = datetime.now(UTC)
```

Comment:
Noted, will resolve this thread if you think it’s ok in this instance (which I also think it should be, since we’re not trying to put this object in the database or convert it to local time or anything)